src/common/settings.js

```diff
@@ -451,6 +451,12 @@ const character_settings = {
         "description": "Unleash your divine soul to deal extra radiant damage equal to your level.",
         "type": "bool",
         "default": false
+    },
+    "indomitable-might": {
+        "title": "Barbarian: Indomitable Might",
+        "description": "Your strength is unmatched!",
```

Comment:
Lol. Not sure if it's worth explaining (summarily) what it does. I do like this description though! :)

Reply:
Removed setting (per below)